src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -3479,16 +3479,39 @@ public void forceTerminateAllRepairSessions() {
         ActiveRepairService.instance.terminateSessions();
     }
 
+    @Deprecated
```

Comment:
`@Deprecated` is very costly to users, so I am not in favor of asking users to migrate to a new method to fix a bug... we should just fix the bug and remove `@Deprecated`